test/library/standard/Path/Prachi-Muskaan/testcases_expandUser.good

```diff
@@ -0,0 +1,5 @@
+/home/user/
```

Comment:
The prediff will generate this file and the cleanfiles will remove it, so you don't want to commit it to the repository